test/gie/more_builtins.gie

```diff
@@ -866,5 +866,54 @@ accept      4.5        52.5        0
 expect      5.875      55.375      0
 -------------------------------------------------------------------------------
 
+-------------------------------------------------------------------------------
+Some tests from adams_ws2
```

Comment:
Why not put these in `test/gie/adams_ws2.gie`?

Reply:
Because of "This gie file was automatically generated using libproject"

Reply:
Oh right, feel free to ignore that. The file can't be reproduced anyway since I generated semi-random numbers for the input coordinates. I just wanted to highlight that the test coordinates didn't come from some "official" source. Maybe I should have phrased that line differently.

Reply:
ok, I'll change its header and move my tests there